src/noos_pyk/clients/json.py

```diff
@@ -34,7 +33,7 @@ def _deserialize(self, response: requests.Response) -> Json:
 def _deserialize_json_response(
     response: requests.Response, valid_content_type: str = DEFAULT_JSON_CONTENT_TYPE
 ) -> Json:
-    content_type, _ = cgi.parse_header(response.headers.get("content-type", ""))
+    content_type = response.headers.get("content-type", "")
```

Comment:
⚠️ 

Worth to be tested as I don't think such a change will have the expected behaviour, ie checking the Json content of an Http response.

```python
In [1]: import requests

In [2]: import cgi

In [3]: r = requests.get('https://api.github.com')

In [4]: ct = r.headers.get("content-type")

In [5]: ct
Out[5]: 'application/json; charset=utf-8'

In [6]: cgi.parse_header(ct)
Out[6]: ('application/json', {'charset': 'utf-8'})

In [7]:
```

Reply:
cgi will be removed in python 3.13. I suggest we add our own `parse_header` function and add more tests